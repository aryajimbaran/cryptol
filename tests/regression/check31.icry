:l check31.iso8859.cry
:l check31.utf8.cry
