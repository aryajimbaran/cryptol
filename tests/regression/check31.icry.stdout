Loading module Cryptol

[error] IO error while loading file: ./check31.iso8859.cry:
    ./check31.iso8859.cry: hGetContents: invalid argument (invalid byte sequence)
Loading module Cryptol
Loading module Fοο
